[2023-05-11T17:57:33.459+0000] {warnings.py:110} WARNING - /usr/local/lib/python3.7/site-packages/***/models/taskinstance.py:871: SAWarning: TypeDecorator ExecutorConfigType() will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
  ti: TaskInstance | None = qry.with_for_update().one_or_none()

[2023-05-11T17:57:33.539+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: job_1_tranvietcuong.file_sensor manual__2023-05-11T17:57:31.424212+00:00 [queued]>
[2023-05-11T17:57:33.597+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: job_1_tranvietcuong.file_sensor manual__2023-05-11T17:57:31.424212+00:00 [queued]>
[2023-05-11T17:57:33.598+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-11T17:57:33.599+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-05-11T17:57:33.599+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-11T17:57:33.718+0000] {taskinstance.py:1383} INFO - Executing <Task(FileSensor): file_sensor> on 2023-05-11 17:57:31.424212+00:00
[2023-05-11T17:57:33.743+0000] {standard_task_runner.py:54} INFO - Started process 862 to run task
[2023-05-11T17:57:33.766+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'job_1_tranvietcuong', 'file_sensor', 'manual__2023-05-11T17:57:31.424212+00:00', '--job-id', '195', '--raw', '--subdir', 'DAGS_FOLDER/tranvietcuong.py', '--cfg-path', '/tmp/tmpnwkgflmq']
[2023-05-11T17:57:33.769+0000] {standard_task_runner.py:83} INFO - Job 195: Subtask file_sensor
[2023-05-11T17:57:33.774+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/tranvietcuong.py
[2023-05-11T17:57:34.392+0000] {task_command.py:384} INFO - Running <TaskInstance: job_1_tranvietcuong.file_sensor manual__2023-05-11T17:57:31.424212+00:00 [running]> on host 09877495cfa0
[2023-05-11T17:57:34.413+0000] {warnings.py:110} WARNING - /usr/local/lib/python3.7/site-packages/***/models/taskinstance.py:873: SAWarning: TypeDecorator ExecutorConfigType() will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
  ti = qry.one_or_none()

[2023-05-11T17:57:34.903+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=user@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=job_1_tranvietcuong
AIRFLOW_CTX_TASK_ID=file_sensor
AIRFLOW_CTX_EXECUTION_DATE=2023-05-11T17:57:31.424212+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-11T17:57:31.424212+00:00
[2023-05-11T17:57:35.017+0000] {base.py:71} INFO - Using connection ID 'fs_default' for task execution.
[2023-05-11T17:57:35.019+0000] {filesystem.py:58} INFO - Poking for file /opt/***/dags/tmp/*dejob*
[2023-05-11T17:57:35.157+0000] {taskinstance.py:1815} INFO - Rescheduling task, marking task as UP_FOR_RESCHEDULE
[2023-05-11T17:57:35.215+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-05-11T17:57:35.405+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-11T17:57:46.888+0000] {warnings.py:110} WARNING - /usr/local/lib/python3.7/site-packages/***/models/taskinstance.py:871: SAWarning: TypeDecorator ExecutorConfigType() will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
  ti: TaskInstance | None = qry.with_for_update().one_or_none()

[2023-05-11T17:57:46.978+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: job_1_tranvietcuong.file_sensor manual__2023-05-11T17:57:31.424212+00:00 [queued]>
[2023-05-11T17:57:47.027+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: job_1_tranvietcuong.file_sensor manual__2023-05-11T17:57:31.424212+00:00 [queued]>
[2023-05-11T17:57:47.027+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-11T17:57:47.028+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-05-11T17:57:47.029+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-11T17:57:47.119+0000] {taskinstance.py:1383} INFO - Executing <Task(FileSensor): file_sensor> on 2023-05-11 17:57:31.424212+00:00
[2023-05-11T17:57:47.133+0000] {standard_task_runner.py:54} INFO - Started process 873 to run task
[2023-05-11T17:57:47.146+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'job_1_tranvietcuong', 'file_sensor', 'manual__2023-05-11T17:57:31.424212+00:00', '--job-id', '196', '--raw', '--subdir', 'DAGS_FOLDER/tranvietcuong.py', '--cfg-path', '/tmp/tmpm7ek8g3c']
[2023-05-11T17:57:47.148+0000] {standard_task_runner.py:83} INFO - Job 196: Subtask file_sensor
[2023-05-11T17:57:47.151+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/tranvietcuong.py
[2023-05-11T17:57:47.426+0000] {task_command.py:384} INFO - Running <TaskInstance: job_1_tranvietcuong.file_sensor manual__2023-05-11T17:57:31.424212+00:00 [running]> on host 09877495cfa0
[2023-05-11T17:57:47.436+0000] {warnings.py:110} WARNING - /usr/local/lib/python3.7/site-packages/***/models/taskinstance.py:873: SAWarning: TypeDecorator ExecutorConfigType() will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
  ti = qry.one_or_none()

[2023-05-11T17:57:47.735+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=user@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=job_1_tranvietcuong
AIRFLOW_CTX_TASK_ID=file_sensor
AIRFLOW_CTX_EXECUTION_DATE=2023-05-11T17:57:31.424212+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-11T17:57:31.424212+00:00
[2023-05-11T17:57:47.813+0000] {base.py:71} INFO - Using connection ID 'fs_default' for task execution.
[2023-05-11T17:57:47.815+0000] {filesystem.py:58} INFO - Poking for file /opt/***/dags/tmp/*dejob*
[2023-05-11T17:57:47.900+0000] {taskinstance.py:1815} INFO - Rescheduling task, marking task as UP_FOR_RESCHEDULE
[2023-05-11T17:57:47.958+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-05-11T17:57:48.142+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-11T17:57:59.088+0000] {warnings.py:110} WARNING - /usr/local/lib/python3.7/site-packages/***/models/taskinstance.py:871: SAWarning: TypeDecorator ExecutorConfigType() will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
  ti: TaskInstance | None = qry.with_for_update().one_or_none()

[2023-05-11T17:57:59.145+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: job_1_tranvietcuong.file_sensor manual__2023-05-11T17:57:31.424212+00:00 [queued]>
[2023-05-11T17:57:59.182+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: job_1_tranvietcuong.file_sensor manual__2023-05-11T17:57:31.424212+00:00 [queued]>
[2023-05-11T17:57:59.182+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-11T17:57:59.183+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-05-11T17:57:59.183+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-11T17:57:59.242+0000] {taskinstance.py:1383} INFO - Executing <Task(FileSensor): file_sensor> on 2023-05-11 17:57:31.424212+00:00
[2023-05-11T17:57:59.254+0000] {standard_task_runner.py:54} INFO - Started process 884 to run task
[2023-05-11T17:57:59.264+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'job_1_tranvietcuong', 'file_sensor', 'manual__2023-05-11T17:57:31.424212+00:00', '--job-id', '197', '--raw', '--subdir', 'DAGS_FOLDER/tranvietcuong.py', '--cfg-path', '/tmp/tmpujz3kcwo']
[2023-05-11T17:57:59.266+0000] {standard_task_runner.py:83} INFO - Job 197: Subtask file_sensor
[2023-05-11T17:57:59.268+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/tranvietcuong.py
[2023-05-11T17:57:59.522+0000] {task_command.py:384} INFO - Running <TaskInstance: job_1_tranvietcuong.file_sensor manual__2023-05-11T17:57:31.424212+00:00 [running]> on host 09877495cfa0
[2023-05-11T17:57:59.539+0000] {warnings.py:110} WARNING - /usr/local/lib/python3.7/site-packages/***/models/taskinstance.py:873: SAWarning: TypeDecorator ExecutorConfigType() will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
  ti = qry.one_or_none()

[2023-05-11T17:57:59.854+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=user@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=job_1_tranvietcuong
AIRFLOW_CTX_TASK_ID=file_sensor
AIRFLOW_CTX_EXECUTION_DATE=2023-05-11T17:57:31.424212+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-11T17:57:31.424212+00:00
[2023-05-11T17:57:59.919+0000] {base.py:71} INFO - Using connection ID 'fs_default' for task execution.
[2023-05-11T17:57:59.921+0000] {filesystem.py:58} INFO - Poking for file /opt/***/dags/tmp/*dejob*
[2023-05-11T17:57:59.924+0000] {taskinstance.py:1481} INFO - Snap. Time is OUT. DAG id: job_1_tranvietcuong
[2023-05-11T17:57:59.958+0000] {taskinstance.py:1406} INFO - Marking task as SKIPPED. dag_id=job_1_tranvietcuong, task_id=file_sensor, execution_date=20230511T175731, start_date=20230511T175759, end_date=20230511T175759
[2023-05-11T17:58:00.040+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-05-11T17:58:00.160+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
